docs/simulate/qvm_basic_example.ipynb

```diff
@@ -0,0 +1,432 @@
+    "\n",
+    "If you don't need this level of control, you can also instantiate a QVM with `cirq_google.engine.create_default_noisy_quantum_virtual_machine`, as in [QVM Creation Template](/cirq/simulate/qvm_builder_code). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "pbHCUPLpq5WE"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Choose a processor (\"rainbow\" or \"weber\")\n",
+    "processor_id = \"weber\"  # @param {type:\"string\"}\n",
+    "\n",
+    "# Construct a simulator with a noise model based on the specified processor.\n",
+    "cal = cirq_google.engine.load_median_device_calibration(processor_id)\n",
+    "noise_props = cirq_google.noise_properties_from_calibration(cal)\n",
+    "noise_model = cirq_google.NoiseModelFromGoogleNoiseProperties(noise_props)\n",
+    "sim = qsimcirq.QSimSimulator(noise=noise_model)\n",
+    "\n",
+    "# Create a device from the public device description\n",
+    "device = cirq_google.engine.create_device_from_processor_id(processor_id)\n",
+    "# Build the simulated local processor from the simulator and device.\n",
+    "sim_processor = cirq_google.engine.SimulatedLocalProcessor(\n",
+    "    processor_id=processor_id, sampler=sim, device=device, calibrations={cal.timestamp // 1000: cal}\n",
+    ")\n",
+    "# Package the processor to use an Engine interface\n",
+    "sim_engine = cirq_google.engine.SimulatedLocalEngine([sim_processor])\n",
+    "print(\n",
+    "    \"Your quantum virtual machine\",\n",
+    "    processor_id,\n",
+    "    \"is ready, here is the qubit grid:\",\n",
+    "    \"\\n========================\\n\",\n",
+    ")\n",
+    "print(sim_engine.get_processor(processor_id).get_device())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "TJfN17frwo-0"
+   },
+   "source": [
+    "## **Create** a circuit, **transform** it (to make it executable on Google quantum hardware) and **choose qubits** on the processor. \n",
+    "\n",
+    "The circuit you use needs to be _device ready_, which means it: \n",
+    "- Is comprised of operations from the device's gate set. \n",
+    "- Is applied to qubits that exist on the device. \n",
+    "- Respects the connectivity of qubits on the device.\n",
+    "\n",
+    "Below is an example of a circuit that has the correct topology to be placed on the Weber device, and how it is prepared to be run on the QVM."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "a_arTtfcwqrz"
+   },
+   "source": [
+    "### Create a GHZ state builder circuit\n",
+    "\n",
+    "The generalized [Greenberger–Horne–Zeilinger (GHZ) state](https://en.wikipedia.org/wiki/Greenberger%E2%80%93Horne%E2%80%93Zeilinger_state){:.external} has the form $\\frac{|00..0⟩ + |11..1⟩}{\\sqrt{2}}$ and in this case will be constructed using 17 qubits, with a Hadamard and a sequence of CNOT gates:"
```

Comment:
commas around the prepositional phrase "in this case"

Reply:
Done.